protocols/datagram.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+// TODO:
+// Come up with a better name for this.
+package beavertail;
+
+message DatagramPush {
+  string busID                    = 10;
+  uint64 passengerCount           = 20;
+  double passengerCountConfidence = 30;
```

Comment:
Do Bryan and KJ know how this is being calculated?

Reply:
cc: @kev231998 and Brian